beta/src/content/learn/separating-events-from-effects.md

```diff
@@ -796,6 +796,22 @@ With `useEvent`, there is no need to "lie" to the linter, and the code works as
 
 <Sandpack>
 
+```json package.json hidden
+{
+  "dependencies": {
+    "react": "experimental",
+    "react-dom": "experimental",
```

Comment:
These should probably use exact versions. Are other Sandpacks using these unstable dist tags as well?

Reply:
Yes, I just did it the same way as other Sandpacks.

Reply:
Gotcha. Then this can stay and we'll discuss usage of this separately.